src/webgpu/api/operation/vertex_state/correctness.spec.ts

```diff
@@ -595,7 +601,23 @@ struct VSOutputs {
       });
     }
 
-    // Run the test shader.
+    return vertexBuffers;
+  }
+
+  runTest(
+    buffers: VertexLayoutState<{ vbOffset?: number }, { shaderComponentCount?: number }>,
+    // Default to using 20 vertices and 20 instances so that we cover each of the test data at least
+    // once (at the time of writing the largest testData has 16 values).
+    vertexCount: number = 20,
+    instanceCount: number = 20
```

Comment:
These also appear to never be used, but if OK if you intend to use them later

Reply:
I'm 90% sure it will be used later.